### PR TITLE
git: Depends on expat and tcl-tk (close #369)

### LINKED ISSUE #369: git: Depends on expat and tcl-tk
`brew install git` fails with the following error:

```
http.h:123: error: ‘CURLE_HTTP_NOT_FOUND’ undeclared (first use in this function)
http.h:125: error: ‘CURLE_FTP_COULDNT_RETR_FILE’ undeclared (first use in this function)
http.h: At top level:
http.h:209: error: expected specifier-qualifier-list before ‘CURLcode’
make: *** [http-fetch.o] Error 1
```

Full debug logs here: https://gist.github.com/db4e9f0a26be39aa0080

### PR DESCRIPTION
**Pull request by iggyvolz**
close #369

### CODE REVIEW

File: Library/Formula/git.rb [hunk 1]
```diff
@@ -33,6 +33,9 @@ class Git < Formula
   depends_on "openssl" if build.with? "brewed-openssl"
   depends_on "curl" if build.with? "brewed-curl"
   depends_on "go" => :build if build.with? "persistent-https"
+  depends_on "expat"
+  depends_on "tcl-tk"
+
```
Comment (sjackman) [inline Library/Formula/git.rb]: Please add `unless OS.mac?` to these two lines.

Comment (iggyvolz) [inline Library/Formula/git.rb]: Whoops!

Comment (iggyvolz) [inline Library/Formula/git.rb]: Fixed.

**Comment by sjackman** [comment]
Thanks, Iggy.

**Pull request closed without merging**